protocol/x/vault/keeper/msg_server.go

```diff
@@ -5,12 +5,12 @@ import (
 )
 
 type msgServer struct {
-	Keeper
+	Keeper types.VaultKeeper
```

Comment:
was Keeper previously untyped?

Reply:
It was previously an embedded struct.

Question: Any reason we chose to stop embedding `Keeper` type into the struct?

Reply:
stopped embedding to be able to mock `x/vault`'s msgServer in unit tests

Reply:
going to remove mock testings as they don't do much and revert back to embedded struct